languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10084,6 +10084,68 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+      <rule id="ADJ_F_ET_ADJ_F" name="adj f et adj f">
```

Comment:
This is a nice rule because it covers a basic error. Using unification and the AdvancedSynthesizerFilter, you can cover all combinations of masculine/feminine as well as singular/plural at once.

Reply:
Indeed, unification and ASF are very useful ! Besides the adjectives interfere with the verb after them as in "Des personnes grandes et intelligentes arrive vers nous." I think it could be look into.